enclave-tls/Makefile

```diff
@@ -0,0 +1,19 @@
+TOPDIR := $(shell readlink -f .)
+
+export TOPDIR
+
+include common.mk
+
+deps := libenclave_tls.so
+
+all: $(deps)
+
+libenclave_tls.so:
+	make -C $(SRCDIR) all
+
+install:
+	make -C $(SRCDIR) install
+
+clean:
+	make -C $(SRCDIR) clean
+	rm -rf $(ENCLAVE_TLS_PREFIX)
```

Comment:
Nope. `make install` installs the binaries to $(ENCLAVE_TLS_PREFIX), so its reverse operation should be `make uninstall`. `make clean` just cleans up local build environment.